### PR TITLE
Updated Json with my little name

### PR DESCRIPTION
**Pull request by Xunjin**

### CODE REVIEW

File: mentors/csharp/v.json [hunk 1]
```diff
@@ -7,4 +7,14 @@
     "avatar_url": null,
     "bio": "I've been using C# professionally for 8 years and love its versatility!"
   }
+],
```
Comment (rpottsoh) [inline mentors/csharp/v.json]: The comma at the end of this line should be at the end of the previous line and this line should be deleted.

```suggestion
```

File: mentors/csharp/v.json [hunk 2]
```diff
@@ -7,4 +7,14 @@
...
+[
```
Comment (rpottsoh) [inline mentors/csharp/v.json]: Please remove this line.
```suggestion
```

**Comment by rpottsoh** [comment]
@Xunjin do you need any assistance or have any questions about the changes needed?  Don't hesitate to ask for help.

**Comment by SleeplessByte** [comment]
Pinging @Xunjin

**Comment by Xunjin** [comment]
Currently i'm being active as mentor C# as i would like, so will close the PR, sorry for the trouble.
@SleeplessByte @rpottsoh

**Comment by kotp** [comment]
We understand @Xunjin and no worries.  We will be here ready when you are!

**Comment by rpottsoh** [comment]
No worries.

**Pull request closed without merging**